api/v6/report_server.thrift

```diff
@@ -225,6 +225,13 @@ struct CompareData {
   2: DiffType  diffType
 }
 
+// This type is used to get line content information for the given file at the
+// given positions.
+struct SourceLine {
+  1: i64        fileId,
+  2: list<i64>  lines
```

Comment:
Maybe a set is better here?

Reply:
Do we want only get some of the lines of the files or we might also want to query ranges if lines? (like 2-5, 30-40, ...)

Reply:
I don't think we would like to get ranges of lines. We use it only at local diff to print the line of resolved bugs.